_layouts/default.html

```diff
@@ -14,11 +14,43 @@
     {% include skipnav.html %}
     {% include header.html %}
 
+    {% assign hero = page.hero %}
+    {% include components/hero.html %}
+
+    {% if page.tagline and page.intro %}
+    <section class="usa-grid usa-section">
+      <div class="usa-width-one-third">
+        <h2>{{ page.tagline }}</h2>
+      </div>
+      <div class="usa-width-two-thirds">
+        {{ page.intro | markdownify }}
+      </div>
+    </section>
+    {% endif %}
+
+    {% capture _graphics %}
+      {% include graphic-list.html graphics=page.graphics %}
+    {% endcapture %}
+
+    {% if page.graphics_position != 'after' %}
+      {{ _graphics }}
+    {% endif %}
+
     <main id="main-content"{% for _attr in layout.main %} {{ _attr[0] }}="{{ _attr[1] }}"{% endfor %}>
-    {{ content }}
+      <div class="usa-section">
```

Comment:
Adding `usa-section` and `usa-grid` to all pages will have an impact on the other layouts. The `page` layout for example already has those classes include in the `<main>` tag (https://github.com/18F/uswds-jekyll/blob/master/_layouts/page.html#L4) so it will double the existing padding surrounding the content area.